src/components/theme/View/ImageView.jsx

```diff
@@ -26,10 +28,24 @@ const ImageView = ({ content }) => (
     {content.description && (
       <p className="documentDescription">{content.description}</p>
     )}
-    <img
-      alt={content.title}
-      src={flattenToAppURL(content.image.scales.preview.download)}
-    />
+    <a href={flattenToAppURL(content.image.download)}>
+      <img
+        alt={content.title}
+        src={flattenToAppURL(content.image.scales.preview.download)}
+      />
+      <figcaption class="discreet">
```

Comment:
``className``

Reply:
Oh, sure. In fact I will remove it, I copied it over the Plone one, and I guess it's no use here anyways.